corehq/apps/locations/dbaccessors.py

```diff
@@ -1,13 +1,27 @@
 from corehq.apps.users.models import CommCareUser
 
 
-def get_users_by_location_id(location_id):
-    """
-    Get all users for a given location
-    """
-    return CommCareUser.view(
+def _users_by_location(location_id, include_docs=True, wrap=True):
+    view = CommCareUser.view if wrap else CommCareUser.get_db().view
+    return view(
         'locations/users_by_location_id',
         startkey=[location_id],
         endkey=[location_id, {}],
-        include_docs=True
-    ).all()
+        include_docs=include_docs,
+    )
+
+
+def get_users_by_location_id(location_id, wrap=True):
```

Comment:
It sounds reasonable to me, I was thinking about that when I made this - you can see I used 3 different functions which each wrap the same private function.  I'll go ahead and do the same for this parameter.